apps/src/music/progress/MusicValidator.ts

```diff
@@ -54,6 +56,9 @@ export default class MusicValidator extends Validator {
     // Get number of different sounds that have been started.
     let playedNumberDifferentSounds = 0;
 
+    // A list of unique invocated ids associated with played trigger sounds.
+    const playedTriggerSoundUniqueInvocationIds = [] as number[];
```

Comment:
nit: we can do the type assignment on the left hand side to avoid the cast, i.e. `const playedTriggeredSoundUniqueInvocationIds: number[] = []`